build/index.js

```diff
@@ -325,13 +331,7 @@ function buildOnce () {
 }
 
 function startServer () {
-  liveServer.start({
-    port: 9000,
-    root: 'target',
-    file: '404/index.html',
-    wait: 50,
-    open: false
-  })
+  require('child_process').spawn('dev_appserver.py', ['--port=9000', '--host=0.0.0.0', 'target/app.yaml'], { stdio: 'inherit' })
```

Comment:
Pro: works the same as the apps in gcloud
Con: we lose live-reload (although it does detect file changes) + lots of extra logging